app/controllers/declaration/PackageInformationController.scala

```diff
@@ -77,24 +77,26 @@ class PackageInformationController @Inject()(
     implicit request: JourneyRequest[AnyContent]
   ): Future[Result] =
     MultipleItemsHelper
-      .saveAndContinue(boundForm, cachedData, isMandatory = true, PackageInformation.limit)
+      .saveAndContinue(boundForm, cachedData, isMandatory(request), PackageInformation.limit)
       .fold(
         formWithErrors => Future.successful(BadRequest(packageInformationPage(mode, itemId, formWithErrors, cachedData))),
         updatedCache =>
           updateExportsCache(itemId, updatedCache)
             .map(_ => navigator.continueTo(mode, nextPage(itemId, request.declarationType)))
       )
 
+  private def isMandatory(journeyRequest: JourneyRequest[_]): Boolean = journeyRequest.declarationType != CLEARANCE
```

Comment:
You could make the request implicit here and not have to pass it in

Reply:
I did that initially, but then realized that it is easier for me to read it as a method when I have these parenthesis present.